guide/onboarding/creating-a-new-wallet.md

```diff
@@ -0,0 +1,91 @@
+
+Instead, a bitcoin wallet contains the "private keys" to your bitcoin. The bitcoin is located on a network that contains a public ledger similar in many ways to a spreadsheet with a record of everyone's transactions and balances (so think of bitcoin wallets more like keychains).
+
+ Like regular keys on a keychain, these keys can be copied. For example, if you gave someone a copy of your house keys, they can unlock the door to your home. Likewise, if someone has a copy of your private key, they can access your funds. This makes [private key management]({{ '/guide/private-key-management/introduction' | relative_url }}) one of the most important considerations of any bitcoin owner.
+
+There are a few different types of wallets that let you send, receive, and store bitcoin.
+
+{% include /tip/open.html label="Tip: Break onboarding up" icon="info" color="blue" %}
+
+The wallet creation process is an excellent place to break things up into different activities, events, and tasks. For example, you might want to dedicate multiple screens when generating a [recovery phrase]({{ '/guide/glossary/#recovery-phrase' | relative_url }}) to explain why it is essential.
+
+{% include /tip/close.html %}
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/onboarding/creating-a-new-wallet/onboarding-welcome-screen.png"
+   retina = "/assets/images/guide/onboarding/creating-a-new-wallet/onboarding-welcome-screen@2x.png"
+   width = 250
+   height = 250
+   alt-text = "Onboarding welcome screen"
+   layout = "float-left-desktop -background -shadow"
+%}
+
+After downloading a wallet application, users are typically presented with the option of either creating a new wallet or restoring an existing one. Obviously, people who have not used bitcoin before will be going through the flow of creating a new wallet.
+
+Once an individual selects “create a new wallet”, it’s typical for beginner-friendly applications to first present informational carousels that provide a high-level overview of the implications, benefits, and responsibility that comes with having a [non-custodial]({{ '/guide/glossary/#non-custodial--custodial-wallet' | relative_url }}) wallet. This moment is a great place to explain that, unlike traditional centralized financial products, a user’s “account” information cannot be recovered by the respective company. Rather, it is up to the user to safely protect their wallet information.
+
+</div>
+
+{% include /tip/open.html label="Don’t hide the implications of a non-custodial wallet" icon="forbid" color="red" %}
+
+Within your onboarding process, be clear about what it means to have a non-custodial wallet and the responsibility that they must accept. Within your onboarding process, be clear that your respective team cannot recover "account" information.
```

Comment:
```suggestion
Within your onboarding process, be clear about what it means to have a self-custody wallet and the responsibility that they must accept. Within your onboarding process, be clear that your respective team cannot recover "account" information.
```

Reply:
Would we recommend that the backup process happens during onboarding? What about wallets that don't do this? Maybe mention some details about this and the tradeoffs of both options.

Reply:
Totally agree, there is so much more depth to go in here. However try view this through the lens of the absolute beginner who we want to give some foundational knowledge to. We will add the details as the guide evolves. Might be better to approach what you suggested in seperate PR.